curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/65f656d81d95713e0a52c927.md

```diff
@@ -0,0 +1,265 @@
+---
+id: 65f656d81d95713e0a52c927
+title: Step 116
+challengeType: 0
+dashedName: step-116
+---
+
+# --description--
+
+To access the name of the third monster from the `monsters` array, you can use the `fighting` variable as an index followed by the `name` property. You should use dot notation to access the property.
+
+```js
+arrayName[index].propertyName
+```
+
+Update your console statement to log the `name` of the third monster from the `monsters` array. Open up the console to see the results.
+
+# --hints--
+
+Your console statement should have the value of `monsters[fighting].name`.
+
+```js
+assert.include(code, 'console.log(monsters[fighting].name)');
```

Comment:
I was hoping to avoid adding more regex tests if possible because we have been receiving a lot of flack for the number of regex tests in the curriculum as is. 

So if there is a way to use another assert method and trim the possible space around the `monsters[fighting].name` value that would be preferable.